Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.12-alpine
```

Comment:
Consider using a specific version of the base image (e.g., `python:3.12.0-alpine`) instead of `python:3.12-alpine` to ensure consistency and avoid unexpected changes when the base image is updated.